src/models/AbstractEntity.php

```diff
@@ -635,10 +635,7 @@ private function addToExtendedFilter(string $extendedFilter, array $extendedValu
     private function updateJsonField(string $key, string|array $value): bool
     {
         $Changelog = new Changelog($this);
-        $valueAsString = $value;
-        if (is_array($value)) {
-            $valueAsString = implode(', ', $value);
-        }
+        $valueAsString = is_array($value) ? implode(', ', $value) : $value;
```

Comment:
had to do this: https://github.com/elabftw/elabftw/blob/cb50b2ca813323621d2f62ad4b98ce02a904fbca/src/models/AbstractEntity.php#L638-L646

psalm wouldn't shut up

Reply:
I used a one-liner:
https://github.com/elabftw/elabftw/commit/2999432ca462079006b0a9fde4cef07275fdf744

Reply:
and psalm doesn't complain??